earthpy/spatial.py

```diff
@@ -441,7 +460,9 @@ def plot_rgb(arr, rgb=(0, 1, 2),
         mask = ~(np.ma.getmask(rgb_bands[0])) * 255
 
         # Add the mask to the array (ise earthpy bytescale)
-        rgb_bands = np.vstack((bytescale(rgb_bands), np.expand_dims(mask, axis=0))).transpose([1, 2, 0])
+        rgb_bands = np.vstack((bytescale(rgb_bands),
+                               np.expand_dims(mask, axis=0))).\
+            transpose([1, 2, 0])
```

Comment:
A comment here to explain this magic number sequence `[1, 2, 0]` would be very useful!

Reply:
do you think using this https://github.com/mapbox/rasterio/blob/147df8a2356d6c59188eeaa621e7e67b8a764e31/rasterio/plot.py#L184 `reshape_as_image()` function from rasterio would make the code better / more expressive vs a comment? i've been thinking about doing that anyway. although it's pretty clean to just do a .transpose but not expressive at all.

Reply:
Either way seems fine to me, and less mysterious!

Reply:
added a comment for the time being!!